src/mpl/include/mpl_atomic_by_lock.h

```diff
@@ -12,128 +12,133 @@
 #include <pthread.h>
 
 /* defined in mpl_atomic.c */
-extern pthread_mutex_t MPL_emulation_lock;
+extern pthread_mutex_t MPLI_emulation_lock;
 
-#define MPL_ATOMIC_IPC_SINGLE_CS_ENTER()         \
-    do {                                         \
-        pthread_mutex_lock(&MPL_emulation_lock); \
+#define ATOMIC_IPC_SINGLE_CS_ENTER()              \
```

Comment:
How about `MPLI_ATOMIC_CS_ENTER`? I don't quite get the meaning of `IPC` and `SINGLE`.

Reply:
Thanks! That's true. I will rename it.